package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "branch-sdk",
-	"version": "2.73.0",
+	"version": "4.0",
```

Comment:
@JagadeeshKaricherla-branch this says 4.0. This can't be right,  when the previous version is 2.73.0

Reply:
@yli-branch fixed the typo.Thanks.